build.gradle.kts

```diff
@@ -16,8 +16,7 @@ buildscript {
     }
 }
 plugins {
-    buildSrcVersions
-    detekt
+  id("io.gitlab.arturbosch.detekt") version "1.1.1"
```

Comment:
@jmfayard Shouldn't the version here be replaced with an entry for the corresponding `plugin.xxx` in `versions.properties`?